graph/graph_windows.go

```diff
@@ -155,6 +182,50 @@ func (graph *Graph) TarLayer(img *Image) (arch archive.Archive, err error) {
 		// We keep this functionality here so that we can still work with the VFS
 		// driver during development. VFS is not supported (and just will not work)
 		// for Windows containers.
-		return graph.driver.Diff(img.ID, img.Parent)
+		root := graph.imageRoot(img.ID)
+		mFileName := filepath.Join(root, tarDataFileName)
+		mf, err := os.Open(mFileName)
```

Comment:
@swernli - This comes back to the (current) Windows graphdriver as merged. The implementations between Linux and the (production) [bearing in mind vfs is only supported for developing the docker daemon, not for running Windows containers] are quite different. The only similarity is the VFS path calling driver.Diff(). Compare  https://github.com/docker/docker/blob/master/graph/graph_unix.go#L113 and https://github.com/docker/docker/blob/master/graph/graph_windows.go#L158. Note that the Windows filter driver at https://github.com/docker/docker/blob/master/graph/graph_windows.go#L140 does quite different actions.